browser-es6.js

```diff
@@ -21,5 +21,12 @@ module.exports = merge(base, {
   },
   plugins: [
     'react'
-  ]
+  ],
+  rules: {
+    'new-cap': 0,
+    'no-extra-parens': 0,
+    'no-inline-comments': 0,
+    'react/jsx-uses-react': [ 2, { pragma: 'dom' } ],
+    'space-before-function-paren': [ 2, { anonymous: 'always', named: 'never' } ]
```

Comment:
Haha, this is one of those places where I definitely prefer this style over what we currently do.

I guess we should have some sort of process in place to propose changes, I'll make a separate PR just for this.